src/main/java/io/stargate/sgv2/jsonapi/api/model/command/Command.java

```diff
@@ -52,4 +56,13 @@
   @JsonSubTypes.Type(value = UpdateManyCommand.class),
   @JsonSubTypes.Type(value = UpdateOneCommand.class),
 })
-public interface Command {}
+public interface Command {
+  /**
+   * Default method if (and only if) there is no "options" property. Used for allowing all Commands
+   * to take in empty Options without having to add Record field if none accepted.
+   */
+  @JsonProperty("options")
+  default void options(Map<String, Object> optionsIgnored) {
```

Comment:
I am not sure I like this option.. This looks very confusing and I dont even understand what does it mean.. now all sub classes have this method and looks strange, especially in the classes that do have the options field.. I am not really pro this change..

Reply:
But the classes that do have options are not overriding this method because they are records and don't even have this setter method.. and even if there would be a setter method, it would still not override as signatures would be different